pkg/services/publicdashboards/database/database.go

```diff
@@ -192,6 +192,17 @@ func (d *PublicDashboardStoreImpl) Update(ctx context.Context, cmd SavePublicDas
 	return err
 }
 
+func (d *PublicDashboardStoreImpl) Delete(ctx context.Context, userOrgId int64, dashboardUid string, uid string) error {
```

Comment:
uid's are unique to a public dashboard so we don't actually need dashboardUid.

```suggestion
func (d *PublicDashboardStoreImpl) Delete(ctx context.Context, orgId int64, uid string) error {
```

Reply:
yes, that was just to be more exact in what we are deleting, taking advantage of we're already receiving the dashboardUid because of permissions check

Reply:
I'll change it